com.woltlab.wcf/package.xml

```diff
@@ -51,6 +51,8 @@
 	</instructions>
 	
 	<instructions type="update" fromversion="5.3.*">
+		<instruction type="script" run="standalone">acp/update_5.4_checkOwnerGroup.php</instruction>
```

Comment:
```suggestion
		<instruction type="script" run="standalone">acp/update_com.woltlab.wcf_5.4_checkOwnerGroup.php</instruction>
```

The script violates our naming schema which should always include the package name.

Reply:
I rebased this branch onto `master`, because there were already erroneous scripts that I just fixed: https://github.com/WoltLab/WCF/commit/2c65abee78283904899420da92f44ab657e75ac8